src/makeCalculator.js

```diff
@@ -38,6 +38,40 @@
  */
 function makeCalculator() {
   // write code here
+
+  const calculator = {
+    result: 0,
+
+    add(number) {
+      return calculator.result + number;
```

Comment:
```suggestion
      return this.result + number;
```

Reply:
Do not pass tests. I tried everything, just approve please

Reply:
try 
this.result += number;
return this;